External/imgui.ini

```diff
@@ -24,7 +24,12 @@ Size=300,1024
 Collapsed=0
 
 [Properties]
-Pos=1520,0
-Size=400,1017
+Pos=624,0
+Size=400,624
+Collapsed=0
+
+[Lighting]
+Pos=624,624
+Size=400,400
 Collapsed=0
 
```

Comment:
no, but is not important, we can commit or not the file. We still need to add it to the .gitignore